tests/testthat/test_pairwise.R

```diff
@@ -180,7 +180,11 @@ test_that("test do_dist with cmd_scale", {
 
   expect_true(ncol(result_kv) %in% c(4,3))
   expect_equal(ncol(result_cols), 4)
-  expect_equal(result_kv[[2]], c(-22.045408, -7.348469, 7.348469, 22.045408), tolerance=0.01)
+
+  # this expectation used to be like following, but added abs to allow results with flipped sign,
+  # which is also correct. For some reason this happens on windows 32bit.
+  # expect_equal(result_kv[[2]], c(-22.045408, -7.348469, 7.348469, 22.045408), tolerance=0.01)
+  expect_equal(abs(result_kv[[2]]), c(22.045408, 7.348469, 7.348469, 22.045408), tolerance=0.01)
```

Comment:
Should the output be stable on any platform? If I draw charts, it will look different on win32 and others, right?

Reply:
Yes, but it's like mirror image and distance between points are kept.